tests/conftest.py

```
@@ -1,38 +1,38 @@
 
 @pytest.fixture
 def setup_example_test(monkeypatch):
-    '''
+    """
     This file is cloned from Silicon Compiler.  We follow its convention
     for organizing CI tests with pytest, so that all testing can be done with
     similar efficiency.  See Silicon Compiler documentation for details
-    '''
+    """
+
     def setup(dir):
```

Ruff (0.15.7)

[error] Function argument `dir` is shadowing a Python builtin

(A002)